src/app/demo/demo.component.scss

```diff
@@ -0,0 +1,198 @@
+@import "src/theme/theme";
+@import "src/theme/theme-variables";
+@import "src/theme/theme-buttons";
+
+@import "src/theme/select";
+.freeze_card{
+font-family: MarkPro;
+font-size: 20px;
+}
+.confirm{
+  width: 398px;
+  height: 70px;
+  font-family: MarkPro;
+  font-size: 28px;
+  font-weight: 500;
```

Comment:
you need not to add all styles, some would come as default. Inspect in the browser and apply style only if you have different values. Please change in all the places